src/app/components/navigation/ErrorClearingRouter.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ReactNode, useEffect, useState } from "react";
+import { useLocation } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import { clearError } from "../../state"
+
+interface ErrorClearingRouterProps {
+    children: ReactNode;
+  }
```

Comment:
Small one, but is this a sign that ESLint is not working (or some rules are deactivated)? There are a couple of spaces before the closing curly bracket, there are also 4 spaces before `children` when the rest of the file seems to have an indentation of 2 spaces.

Reply:
Yep we've still not got ES lint fully setup and configured, nor have we cleared out the ~600 existing ESLint errors that flag up. 

Related tickets:
Linting errors: https://github.com/isaaccomputerscience/isaac-cs-issues/issues/8
Linting config: https://github.com/isaaccomputerscience/isaac-cs-issues/issues/27
Prettier: https://github.com/isaaccomputerscience/isaac-cs-issues/issues/28

I've just reformatted this file though so should be fixed